tuf/client_rework/updater_rework.py

```diff
@@ -35,27 +36,26 @@ class Updater:
     """
     Provides a class that can download target files securely.
 
-    Attributes:
-        metadata:
-
-        repository_name:
-
-        mirrors:
-
-        fetcher:
-
-        consistent_snapshot:
+    TODO
     """
 
     def __init__(
         self,
         repository_name: str,
-        repository_mirrors: Dict,
+        metadata_url: str,
+        default_target_url: Optional[str] = None,
```

Comment:
nit: these are prefixes or bases, not full urls – should we call them `metadata_base_url` and `default_target_base_url`?
Or perhaps `default_base_url` and `targets_base_url` ?